core/src/utils/prove.rs

```diff
@@ -476,6 +476,12 @@ pub(super) mod baby_bear_blake3 {
         }
     }
 
+    impl Clone for BabyBearBlake3 {
```

Comment:
where do we need to clone the config?

Reply:
Right now it's required in prove_with_config, but maybe it's not necessary?

Reply:
I just removed the bound and it seems fine.